test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -192,7 +192,7 @@
                     "System.Configuration": "4.0.0.0",
                     "System.Console": "4.1.2.0",
                     "System.Core": "4.0.0.0",
-                    "System.Data.Common.Reference": "4.2.2.0",
+                    "System.Data.Common": "4.2.2.0",
```

Comment:
I looked and all of these `.Reference` changes should be okay as they're all part of the framework so not deployed with our package.